membership_policy.md

```diff
@@ -0,0 +1,125 @@
+Defined by: Member of the KubeVirt GitHub organization
+
+### Requirements
+
+  * Enabled [two-factor authentication](https://help.github.com/articles/about-two-factor-authentication) on their GitHub account
+  * Actively contributing to the project with multiple contributions to the project or community. Contribution may include, but is not limited to:
+    * Authoring or reviewing PRs on GitHub
+    * Filing or commenting on issues on GitHub
+    * Contributing to community discussions (e.g. meetings, Slack, email discussion forums, Stack Overflow)
+    * GitHub Insight metrics will be reviewed periodically
+  * Subscribed to kubevirt-dev@googlegroups.com
+  * Has read the [contributor guide](CONTRIBUTING.md)
+  * Actively contributing to the project
+    * Contributions according to GitHub and/or Kubernetes metrics in the past 90 days.
+  * Sponsored by 2 org members. Note the following requirements for sponsors:
+    * Sponsors must have close interactions with the prospective member
+    * Sponsors must be reviewers or approvers in the KubeVirt OWNERS file.
+    * Sponsors should be from multiple member companies to demonstrate integration across community.
+  * Open a PR against the [org members section](https://github.com/kubevirt/project-infra/blob/master/github/ci/prow/files/orgs.yaml#L21)
+    * Ensure your sponsors are @mentioned on the issue
+    * Complete every item on the [checklist](membership_checklist.md)
+    * Make sure that the list of contributions included is representative of your work on the project.
+  * Have your sponsoring org members reply confirmation of sponsorship: +1
+  * Once your sponsors have responded, your request will be reviewed by KubeVirt project approvers. Any missing information will be requested.
+
+## Responsibilities and privileges
+
+  * Responsive to issues and PRs assigned to them
+  * Responsive to mentions
+  * Active owner their contributions (unless ownership is explicitly transferred)
+  * Contribution is well tested
+  * Tests consistently pass
+  * Addresses bugs or issues discovered after contribution is accepted
+  * Members can do /lgtm on open PRs.
+  * They can be assigned to issues and PRs, and people can ask members for reviews with a /cc @username.
+  * Tests can be run against their PRs automatically. No /ok-to-test needed.
+  * Members can do /ok-to-test for PRs that have a needs-ok-to-test label, and use commands like /close to close PRs as well.
+
+!! note
+Members who frequently contribute are expected to proactively perform reviews and work towards becoming a primary reviewer.
+
+## Reviewer
+
+Reviewers are trusted to review contributions for quality and correctness on some part of the project. They are knowledgeable about both the project and engineering principles.
+Defined by: reviewers entry in an OWNERS file of the KubeVirt project.
+Note: Acceptance of contributions requires at least one approver in addition to the assigned reviewers.
+
```

Comment:
Like in the approver case, a `Reviewer Status is scoped to a part of the project.` can be added here as well.